src/DataStructures/VariablesHelpers.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines helper functions for use with Variables class
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Index.hpp"
+#include "DataStructures/SliceIterator.hpp"
+#include "DataStructures/Variables.hpp"
+
+template <size_t>
+class Index;
+
+/*! \ingroup DataStructures
+ *  \brief Slices the data within evolved_vars to a codimension 1 slice in the
+ * sliced_dim dimension. Which surface to slice to is determined by slice_point.
```

Comment:
Yea, it would probably only show the first sentence in Doxygen (though I like the wording)